util/design/gen-rng-health-thresholds.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""This script generates health threshold configuration values for the
+entropy_src block.
+
+The implementation is identical to the `ideal_threshold_recommendation` DV
+utility function available in dv/env/entropy_src_env_pkg.sv.
+
+
+
+The sigma value versus probability of test failure is as follows (taken from
+the reference above):
+
+No. of sigma  | Approximate probability of test failure ( P(x) = 1 - erf( x / sqrt(2) ))
+--------------|----------------------------------
+            1 |  31.7%
+          1.5 |  13.4%
+            2 |   4.6%
+          2.5 |   1.2%
+            3 |  0.27%
+          3.3 |   0.1%
+          3.9 |   1e-4
+         4.42 |   1e-5
+          4.9 |   1e-6
+
+Notes:
+
+`--window_size` must be set to reflect the respective `FIPS_WINDOW` or
+`BYPASS_WINDOW` in bits. The following are window size recommended values:
+
+- FIPS: 2048
+- Boot mode: 384
+
+`--per-bit` must be set to reflect the `RNG_BIT_ENABLE` entropy src configuration.
+
+"""
+import argparse
+import math
+import sys
+
+# Matches the `RNG_BUS_WIDTH` in entropy_src_pkg.sv
+_RNG_BUS_WIDTH = 4
+
+# List of test windows supported by this script.
+_TEST_NAMES = ("ADAPTP", "BUCKET", "MARKOV")
```

Comment:
nit: an enum here might provide a better protection against typos:
```suggestion
class _TestNames(Enum)
  ADAPTP = "ADAPTP"
  BUCKET = "BUCKET"
  MARKOV = "MARKOV"
```

Reply:
Done. Good idea. Switched to enum class.